libres/old_tests/enkf/CMakeLists.txt

```diff
@@ -78,7 +78,6 @@ foreach(
   gen_kw_logarithmic
   enkf_forward_init_GEN_KW
   enkf_forward_init_GEN_PARAM
-  enkf_umask_config
```

Comment:
Should delete the .cpp file also: libres/old_tests/enkf/test_enkf_umask_config.cpp